kv/dist_sender.go

```diff
@@ -425,7 +425,15 @@ func (ds *DistSender) getDescriptors(
 func (ds *DistSender) sendSingleRange(
 	ctx context.Context, ba roachpb.BatchRequest, desc *roachpb.RangeDescriptor,
 ) (*roachpb.BatchResponse, *roachpb.Error) {
-	log.Trace(ctx, fmt.Sprintf("sending RPC to [%s, %s)", desc.StartKey, desc.EndKey))
+	// Hack: avoid formatting the message passed to Span.LogEvent for
```

Comment:
Add a TODO. It can be in my name if you must. I'm also OK with not logging the keys (which are mostly ugly anyway). Maybe log the ReplicaID instead, which should be much cheaper.